### PR TITLE
Change some layout, add rulers

### PR DESCRIPTION
**Pull request by PuruVJ**
## Description
Change some layout, add rulers, bigger font

## Attach Screenshot
image

### CODE REVIEW

**Comment by PuruVJ** [comment]
Closes #633

File: twindle-cli/renderer/pdf/templates/Thread.hbs [hunk 1]
```diff
@@ -7,9 +7,11 @@
 
   body {
     margin: 10px 10px 20px 10px;
+<<<<<<< HEAD
```
Comment (proful) [inline twindle-cli/renderer/pdf/templates/Thread.hbs:10]: Danger!!!

File: twindle-cli/renderer/pdf/templates/Thread.hbs [hunk 2]
```diff
@@ -7,9 +7,11 @@
...
+=======
 
-    font-size: 105%;
-
+>>>>>>> ac0198f9195476189a082f6db263d1f1f24d68c9
```
Comment (proful) [inline twindle-cli/renderer/pdf/templates/Thread.hbs:13]: Danger!!!

Comment (PuruVJ) [reply twindle-cli/renderer/pdf/templates/Thread.hbs]: Fixed

File: twindle-cli/renderer/pdf/templates/Thread.hbs (hunk 1 above)
Comment (PuruVJ) [reply twindle-cli/renderer/pdf/templates/Thread.hbs]: Fixed

**Comment by proful** [comment]
@PuruVJ Danger 😂

image

**Comment by PuruVJ** [comment]
> Closes #633

Doesn't close it fully, the rich content rendering part is still left

**Comment by PuruVJ** [comment]
@proful Check again

**Comment by arthurbuhl** [comment]
> 
> 
> Closes #633

This only works if you mention this in the first description when we create a PR. Please check #167 for more details. Thanks.

**Pull request merged**